### ISSUE TITLE
How to integrate with Express

### ISSUE THREAD

**Issue by amitaibu** [likerRr/code4goal-resume-parser#2]
It seems the current lib listens to a directory and spits out a .json file.

How would you go with having a "proper" RESTful server that would get the CV file and return a JSON?

**Comment by likerRr** [comment]
You're right, within current architecture I'm not sure if it possible. If I understood you correctly you can get any files you want by your express server and then put them into parser's folder. Then just run parser. But in this case you won't have any methods to listen about start/end events. Sorry

**Comment by amitaibu** [comment]
No worries, thanks!
